src/cli/app.js

```diff
@@ -109,6 +110,17 @@ async function mcodeApp(Client, fromDate, toDate, pathToConfig, pathToRunLogs, d
       }
     }
 
+    // check if config specifies that MRN needs to be masked
+    // if it does need to be masked, mask all references to MRN outside of the patient resource
+    const patientConfig = config.extractors.filter((e) => e.type === 'CSVPatientExtractor')[0];
+    if ('constructorArgs' in patientConfig && 'mask' in patientConfig.constructorArgs) {
```

Comment:
```suggestion
    if (patientConfig && ('constructorArgs' in patientConfig && 'mask' in patientConfig.constructorArgs)) {
```

In tandem with above `find` suggestion, might as well guard against the undefined case here. Even though I think the `if` will still work find without it. It's just a little bit more expressive